bin/trento-install-checks

```diff
@@ -30,7 +32,7 @@ if [ -d "$checks_dst" ] && [ -n "$(find "$checks_dst" -maxdepth 0 -type d -empty
   exit $?
 fi
 
-echo "Make sure $checks_dst exists, is empty and accessible" 1>&2
+echo "Make sure $checks_dst is empty and accessible" 1>&2
```

Comment:
I'm actually having second thoughts about this, I think we should just empty the directoty and reinstall the checks every time, but I wanna think about it before making that call